doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2373,6 +2373,30 @@ If you managed to get correct accented characters from shibboleth while this set
 
 .. _:ComputeBaseUrl:
```

Comment:
This ` .. _:ComputeBaseUrl:` should be just above the ` :ComputeBaseUrl` section below. It's an anchor so people can link to it.

Otherwise, the docs look good. You can preview them here: https://dataverse-guide--8880.org.readthedocs.build/en/8880/installation/config.html#shibaffiliationorder